<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistrationComplete</class>
 <widget class="QWidget" name="RegistrationComplete">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>310</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>310</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>310</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>401</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;center&gt;&lt;p&gt;&lt;font color=&quot;green&quot;&gt;&lt;b&gt;Congratulations!&lt;/b&gt; Customer Registration Complete&lt;/font&gt;&lt;/p&gt;&lt;/center&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="dairyNameLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>401</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;center&gt;&lt;p&gt;RD Square Dairy, Lalya ka Bas Jaipur&lt;/p&gt;&lt;/center&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="custLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>381</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
color: black

}</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;p&gt;New customer added&lt;/p&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="usernameLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>401</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;p&gt;&lt;b&gt;User Name: &lt;/b&gt;&lt;/p&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="custIDLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>401</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;p&gt;&lt;b&gt;Customer ID No: &lt;/b&gt;&lt;/p&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="passwordLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;p&gt;&lt;b&gt;Password: &lt;/b&gt;&lt;/p&gt;&lt;/center&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="printButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ready &amp;Print</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>270</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&amp;OK</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="usernameshowLabel">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>120</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
color: blue
}</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;p&gt;user_5436&lt;/p&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="passwordShowLabel">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>160</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
color: blue
}</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;p&gt;7675645345765&lt;/p&gt;&lt;/center&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="custIDShowLabel">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>200</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
color: blue
}</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;p&gt;MP1000001&lt;/p&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="passwordLabel_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>160</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;p&gt;(same as mobile number) &lt;/p&gt;&lt;/center&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="emailLbel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {

color: red
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;p&gt;&lt;center&gt;Message has been sent!&lt;/center&gt;&lt;/p&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>okButton</tabstop>
  <tabstop>printButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
